stockAndPrices/services/app/main.py

```diff
@@ -16,12 +16,39 @@ def home():
 
 @app.route('/login', methods=['POST', 'GET'])
 def login():
+    rtn = ""
+    db_host = request.args.get('dbHost')
+    db_port = request.args.get('dbPort')
+    try:
+        client = MongoClient(db_host, int(db_port))
+        client.server_info()
+    except:
+        rtn = "Connection Error! Please provide valid dbHost and dbPort as query strings and make sure your" \
+              " mongoDB server is running."
+        return f"<h1>{rtn}<h1>"
+
+    sample_db = client.sampleDB
+    dev_col = sample_db.developer
     if request.method == "GET":
-        return "<h1>Login Page</h1>"
+        if dev_col.find_one({"first": "Hello"}) is None:
+            dev_col.insert_one({"first": "Hello", "last": "World"})
```

Comment:
Since this is a get request, let us not insert data into the database. Instead if the data is None, lets tell the user there is no data instead.